Fix test_read_simple_quarto.py::test_qmd_to_ipynb

The test fails on the [recent runs](https://github.com/mwouts/jupytext/runs/4361191093?check_suite_focus=true).
We should preserve the error message and fix the issue.

I have improved the error display with https://github.com/mwouts/jupytext/commit/bafecd7d1dac3b9b296b8eade9c6e233b78f923b (still not perfect though) and now I get this:
```
         if proc.returncode:
            raise QuartoError(
>               f"{' '.join(cmd)} exited with return code {proc.returncode}\n"
                f"Quarto output:\n{out}\n"
                f"Quarto errors:\n{err}"
            )
E           jupytext.quarto.QuartoError: quarto convert --log-level warning /tmp/tmpfgvtxk2n.qmd exited with return code 1
E           Quarto output:
E           b''
E           Quarto errors:
E           b'\x1b[91mERROR: Error: strings must be non-empty\x1b[39m\n'
```
@cderv or @jjallaire , do you have any idea what could trigger this error? The text representation of the test notebook is simply this:
https://github.com/mwouts/jupytext/blob/6fee30cd72923de56b3d1ec6375bd5a1947f901a/tests/test_read_simple_quarto.py#L11-L16

--- a/jupytext/quarto.py
+++ b/jupytext/quarto.py
@@ -22,12 +22,13 @@ def quarto(args, filein=None):
     if filein:
         cmd.append(filein)
 
-    proc = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+    proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     out, err = proc.communicate()
     if proc.returncode:
         raise QuartoError(
-            f"{' '.join(cmd)} exited with return code {proc.returncode}\n{err}"
+            f"{' '.join(cmd)} exited with return code {proc.returncode}\n{err.decode('utf-8')}"
         )
+
     return out.decode("utf-8")
 
 

